src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -24,11 +25,13 @@ public class ApiShould : IClassFixture<PlatformSpecificTestContext>, IDisposable
         private SQLSetupStrategy fixture;
 
         private readonly string tableName;
+        private readonly string customKeyStoreProviderTableName;
```

Comment:
NIT: Change it to `_customKeyStoreProviderTableName` to follow the naming convention.